x-pack/plugins/security_solution/public/flyout/document_details/shared/hooks/use_fetch_graph_data.ts

```diff
@@ -36,6 +36,11 @@ export interface UseFetchGraphDataParams {
      * Defaults to true.
      */
     refetchOnWindowFocus?: boolean;
+    /**
+     * If true, the query will keep previous data till new data received.
+     * Defaults to false.
+     */
+    keepPreviousData?: boolean;
```

Comment:
This allows to fetch data and wait for the results before re-rendering the graph.